instrumentation/net/http/otelhttp/handler.go

```diff
@@ -202,6 +206,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	// Add metrics
 	attributes := append(labeler.Get(), semconv.HTTPServerMetricAttributesFromHTTPRequest(h.operation, r)...)
+	h.counters[RequestCount].Add(ctx, 1, attributes...)
```

Comment:
Is this necessary?  Can't the same information be extracted from the count of the server latency histogram?

Reply:
I'm using Dynatrace, at least in their UI I can't seem to find a way to get the count out of the `http.server.duration` metric, is it required to be available by opentelemetry?